code/math/vecmat.h

```diff
@@ -36,6 +36,10 @@
 //Note: NO RETURN VALUE
 #define vm_vec_zero(v) (v)->xyz.x=(v)->xyz.y=(v)->xyz.z=0.0f
 
+//macro to set a matrix to zero.
+//Note: NO RETURN VALUE
+#define vm_mat_zero(m) (vm_vec_zero(&(m)->vec.rvec), vm_vec_zero(&(m)->vec.uvec), vm_vec_zero(&(m)->vec.fvec))
```

Comment:
Get rid of the ampersand here; it should have the same calling syntax as the other macro (`vm_vec_zero(&m)` similar to `vm_vec_zero(&vec)`)

Reply:
vm_vec_zero takes a pointer to a vector so vm_mat_zero needs to send it pointers instead of the vectors themselves.